src/test/java/com/example/rets_api/repository/UtilsTest.java

```diff
@@ -44,14 +47,19 @@ public static RoomEntity createRoomEntity(RoomType roomType){
         room.setLength(5);
         room.setLengthUnit(LengthWidthUnit.METERS);
         room.setWidth(9);
-        room.setWidthUnit(LengthWidthUnit.METERS);
```

Comment:
Why those values where removed? Did you merge/rebase with qa before update this PR? Looks that your branch is not up to date with qa...

Reply:
done